src/otx/core/data/transform_libs/mmdet.py

```diff
@@ -8,21 +8,26 @@
 from typing import TYPE_CHECKING, Callable
```

Comment:
@harimkang @vinnamkim @kprokofi

Please review the specified file, as I've consolidated the instance segmentation MMDet pipeline with the detection's pipeline based on Vinnam's feedback.